src/library-authoring/LibraryBlock/wrap.js

```diff
@@ -0,0 +1,392 @@
+/* eslint-disable no-param-reassign */
+/* istanbul ignore file */
```

Comment:
I'm not sure how we can test this, as a good part of this code relies on the hosted `xblock-bootstrap.html`.

Reply:
Yeah, we can't really test it without actually having the LMS running. So just leave it for now.